core-services/prow/02_config/_boskos.yaml

```diff
@@ -646,6 +646,14 @@ resources:
   - us-west-2--aws-sd-qe-quota-slice-2
   state: free
   type: aws-sd-qe-quota-slice
+- names:
+  - ap-northeast-1
```

Comment:
@yunjiang29 I have question. add _ap-northeast-1--aws-terraform-qe-quota-slice-0_
But when I execute _make boskos-config_ - the string after "1" is removed. What am I doing wrong?